resources/views/tools/database/relationship-partial.blade.php

```diff
@@ -41,8 +41,8 @@
                 <option value="belongsToMany" @if(isset($relationshipDetails->type) && $relationshipDetails->type == 'belongsToMany'){{ 'selected="selected"' }}@endif>Belongs To Many</option>
             </select>
             <select class="relationship_table select2" name="relationship_table_{{ $relationship['field'] }}">
-                @foreach($tables as $table)
-                    <option value="{{ $table }}" @if(isset($relationshipDetails->table) && $relationshipDetails->table == $table){{ 'selected="selected"' }}@endif>{{ ucfirst($table) }}</option>
+                @foreach($tables as $tbl)
```

Comment:
Changing `$table` to `$tbl` doesn't seem necessary.  Was there a reason for this change?

Reply:
$table has the current table name Wich we are talking about on the relationship, if we assign again the $table inside the foreach, later when you echo the $table variable you will have the name of the last table inside the  tables array and no longer the name of the table we are using.

Reply:
ohhhh.... good call.  I was looking solely at the PR changes, so I didn't see that bigger picture.  Very good.  Merging soon